### ISSUE TITLE
Displaying direct and followers-only posts

### ISSUE THREAD

**Issue by tofuwabohu** [wmurphyrd/guppe#34]
Thanks for providing gup.pe. It is a nice tool, especially on instances that aren't well federated.

Currently, guppe accounts share all posts they are mentioned in. I think they should not display posts that are of _public_ or _followers-only_-visibility. Users might want to communicate in private and mention guppe accounts, e.g. to instruct each other on how to use these. Displaying these posts on the guppe's public profile violates these visibility settings.

![image](https://user-images.githubusercontent.com/66644064/102810685-70336700-43c4-11eb-9efe-02c65e4ba874.png)

**Comment by wmurphyrd** [comment]
I appreciate you including a use case - I've been quite perplexed why people want to tag a sharing group with non-public posts. I've removed all post display from the web profile until this can be sorted out, so it is no longer possible to view non-public posts shared with guppe. It still attempts to boost anything you ask it to boost, but those will be ignored in the Mastodon clients if they aren't public.

**Comment by goodevilgenius** [comment]
I disagree. There are certainly cases where I would want to share something publicly, and also with a group.

For example, if a television show I watch announces a new season, I might want to share it publicly, and also with a group for that show.
